odoo/sql_db.py

```diff
@@ -754,13 +669,9 @@ def __init__(self, pool, dbname, dsn):
         self.dsn = dsn
         self.__pool = pool
 
-    def cursor(self, serialized=True):
-        cursor_type = serialized and 'serialized ' or ''
-        _logger.debug('create %scursor to %r', cursor_type, self.dsn)
-        return Cursor(self.__pool, self.dbname, self.dsn, serialized=serialized)
-
-    # serialized_cursor is deprecated - cursors are serialized by default
-    serialized_cursor = cursor
+    def cursor(self, cursor_name='serialized '):
+        _logger.debug('create %scursor to %r', cursor_name, self.dsn)
```

Comment:
What's the purpose of this?  All cursors are serialized anyway.

Reply:
Migration still use the second argument :cry: (https://github.com/odoo/upgrade/blob/4fcc852afe31a9f74dc403dd6a0d693dc1e4673c/migrations/point_of_sale/saas~11.5.1.0.1/post-migrate.py#L106 , https://github.com/odoo/upgrade/blob/3a7090990666f32f198f661eaea848c6c0e141f0/migrations/base/8.0.1.3/post-00-split-filestore.py#L20 ) I am not sure if I can remove it like that.